packages/oui-numeric/README.md

```diff
@@ -0,0 +1,27 @@
+# Numeric
+
+<component-status cx-design="complete" ux="prototype"></component-status>
+
+## Usage
+
+```html:preview
+<oui-numeric id="someId"
+             text="CPU"
+             unit="core"
+             model="$ctrl.cpuCores">
+</oui-numeric>
+```
+
+## API
+
+| Attribute     | Type     | Binding | Values              | Default | Description                         |
```

Comment:
We should have a column for attributes with one-time bindings

See https://github.com/ovh-ux/ovh-ui-angular/tree/feature/oui-button/packages/oui-button (for example)